www/sejd00/staticka-html-prezentace/css/style.css

```diff
@@ -0,0 +1,156 @@
+body, html {
+    height: 100%;
+    margin: 0;
+    font-family: 'Amatic SC', cursive;
+    font-size: 25px;
+}
+
+article.flex-item{
+    font-size: 20px;
+
+}
+
+.main-headline{
+text-align: center;
+}
+
+a {
+    color: white;
+    text-transform: uppercase;
+    text-decoration: none;
+    
+}
+
+nav {
+    border: solid black 1px;
+}
+
+a:active{
+    color: red;
+}
+
+a:hover {
+    color: gray;
+}
+
+.site-description{
+    color: white;
+}
+
+
+.wrap{
+    background-color: rgba(0, 0, 0, 0.3);
+    margin-top: 100px;
+    padding-top: 50px;
+    padding-bottom: 50px;
+    padding-left: 10%;
+    margin-right: 50%;
+}
+
+.bg-main{
+    background-image: url("/4IZ268-2018-2019-ZS/www/sejd00/staticka-html-prezentace/img/cover.jpg");
```

Comment:
staci 'img/cover.jpg'

takhle to nebude fungovat, kdyz se web umisti na jinem serveru